spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
I think adding a NOTE instead of a comment seems reasonable. I don't agree with the logic that "changing anything in ES could break the web security model, so we shouldn't bother documenting anything." We have an explicit ask from an implementer (see linked thread) for such a note to exist in the spec. Documenting such security-impactful pitfalls seems important.